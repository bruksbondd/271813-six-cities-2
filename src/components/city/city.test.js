import React from 'react';
import renderer from 'react-test-renderer';
import City from './city.jsx';

it(`render correctly city`, () => {
  const city = renderer
    .create(<City
      name={`Amsterdam`}
      selectedCity={`Amsterdam`}
      onClick={() => {}}
    />)
    .toJSON();
  expect(city).toMatchSnapshot();
});
