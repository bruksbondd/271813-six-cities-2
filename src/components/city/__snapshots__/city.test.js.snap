// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly city 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
